doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md

```diff
@@ -2,304 +2,181 @@
 
-## Compile C endpoint SDK
+ - CMake (minimum required version is 2.8.12)
+ - C99 compatible compiler (e.g. GCC)
+ - OpenSSL
 
-To build the C endpoint SDK, do the following:
+To install dependencies on Ubuntu execute the following command:
 
-<!-- TODO: Fix this link when Administration-UI-guide#AdministrationUIguide-GeneratingSDK are updated -->
-1. [Generate ](Administration-UI-guide#AdministrationUIguide-GeneratingSDK)the C endpoint SDK in Admin UI.
-2. Download and untar the Kaa C SDK archive.
+```
+sudo apt-get install cmake libssl-dev gcc
+```
 
-   ~~~ bash
-   $ tar xfv kaa-c-ep-sdk.tar.gz
-   ~~~
+### Build procedure
 
-3. Run the following commands.
+After you generated archive with C SDK, proceed as follows:
 
-   ~~~ bash
-   $ mkdir build
-   $ cd build
-   $ cmake ..
-   $ make
-   $ make install
-   ~~~
+1. Unpack SDK:
 
-## Configure C endpoint SDK
+        tar -xvf c-sdk-archive-name.tar.gz
 
-To configure the C endpoint SDK build, you can optionally specify the following parameters for the **cmake** command.
+2. Create the directory where SDK will be built:
 
 
-{% include csv-table.html csv_path='cmake-c-options.csv' table_id='table-c-cmake-options' width='100%' %}
+        mkdir build
+        cd build
 
+3. Configure the build via CMake:
 
-**\* NOTE:**
+        cmake ..
 
-Before running the **cmake** command with the KAA\_PLATFORM parameter for a platform other than supported, do the following:
+4. Perform build:
 
-1. Create a folder in $KAA\_HOME/listfiles/platform/ and name it using \[a-zA-z\_-\] symbols. You will be able to use the folder name as a value for the KAA\_PLATFORM parameter.
-2. Put the CMakeLists.txt file into the created folder. This file may contain specific compilation/linking flags, platform-dependent source files, third-party library dependencies, that is all information necessary for building the C endpoint SDK for this platform.
-3. Optionally, specify the following parameters in the CMakeLists.txt file.
-  * KAA\_INCLUDE\_PATHS - full path(s) to folder(s) containing additional header files
-  * KAA\_SOURCE\_FILES - full path(s) to additional source files
-  * KAA\_THIRDPARTY\_LIBRARIES - third-party libraries (the name of the library, for example, ssl, crypto)
+        make
 
-The following example illustrates the build procedure for the debug build with the INFO log level and disabled EVENTS feature.
+## Building Kaa application
 
-~~~
-mkdir build
-cd build
-cmake -DKAA_DEBUG_ENABLED=1 -DKAA_MAX_LOG_LEVEL=4 -DKAA_WITHOUT_EVENTS=1 ..
-make
-make install
-~~~
+This section is about how to build your Kaa application using C SDK.
 
-# C++ endpoint SDK
+In this guide we use *CMake* as a build system for the application, so focus on the writing the build system rather than the application code. If you want to master writing application code go to the [Programming guide]({{root_url}}/Programming-guide) page. Before continuing, make sure that all [dependencies](#dependencies) are installed.
```

Comment:
It's better to rephrase `as a build system for the application, so focus on the writing the build system`